test/integration/GovernorUpdate.js

```diff
@@ -132,75 +158,169 @@ describe("Governor update", () => {
             await expect(governor.execute(update)).to.be.revertedWith(
                 "new Minter cannot be current Minter"
             )
         })
 
         it("step 1 'migrateToNewMinter' fails: new Minter cannot be null address", async () => {
-            migrateData = minter.interface.encodeFunctionData(
+            const migrateData = minter.interface.encodeFunctionData(
                 "migrateToNewMinter",
                 [ethers.constants.AddressZero]
             )
 
             const update = {
-                target: [migrateTarget, setInfoTarget],
-                value: ["0", "0"],
-                data: [migrateData, setInfoData],
+                target: [
+                    migrateTarget,
+                    transferStateTarget,
+                    grantRoleTarget,
+                    revokeRoleTarget,
+                    setInfoTarget
+                ],
+                value: ["0", "0", "0", "0", "0"],
+                data: [
+                    migrateData,
+                    transferStateData,
+                    grantRoleData,
+                    revokeRoleData,
+                    setInfoData
+                ],
                 nonce: 0
             }
 
-            // run the migrate to new minter update
+            // Run the migrate to new minter update
             await governor.stage(update, "0")
             await expect(governor.execute(update)).to.be.revertedWith(
                 "new Minter cannot be null address"
             )
         })
 
-        it("step 2 'setContractInfo' fails: wrong target", async () => {
-            migrateData = minter.interface.encodeFunctionData(
+        it("step 1 'migrateToNewMinter' fails: new Minter must be registered", async () => {
+            const migrateTarget = newMinter.address
+            const migrateData = minter.interface.encodeFunctionData(
                 "migrateToNewMinter",
-                [newMinter.address]
+                // Minter is currently registered in Controller
+                [minter.address]
             )
+            const update = {
+                target: [
+                    migrateTarget,
+                    transferStateTarget,
+                    grantRoleTarget,
+                    revokeRoleTarget,
+                    setInfoTarget
+                ],
+                value: ["0", "0", "0", "0", "0"],
+                data: [
+                    migrateData,
+                    transferStateData,
+                    grantRoleData,
+                    revokeRoleData,
+                    setInfoData
+                ],
+                nonce: 0
+            }
+
+            // Run the migrate to new minter update
+            await governor.stage(update, "0")
+            await expect(governor.execute(update)).to.be.revertedWith(
+                "new Minter must be registered"
+            )
+        })
 
+        it("step 2 'transferStateData' fails: wrong target", async () => {
```

Comment:
maybe better to use method name on test name
```suggestion
        it("step 2 'migrateOldMinterState' fails: wrong target", async () => {
```